database/src/main/scala/cromwell/database/slick/MetadataComponent.scala

```diff
@@ -82,6 +82,14 @@ trait MetadataComponent {
     } yield m
   )
 
+  val workflowExists = Compiled(
+    (workflowExecutionUuid: Rep[String]) => (for {
+      m <- metadata
+      if m.workflowExecutionUuid === workflowExecutionUuid
+      if m.key === WorkflowMetadataKeys.SubmissionTime
```

Comment:
This is _ok_, but can you explain why you need to check for a key at all? Shouldn't a row in the database with the UUID (line 88) be enough to verify existence?